### PR TITLE
refactor(log): use LOG_WARNING_F instead of LOG_WARNING (3/3)

### PR DESCRIPTION
**Pull request by WHBANG**
### What problem does this PR solve? 
https://github.com/apache/incubator-pegasus/issues/1305

This patch aim to use LOG_WARNING_F instead of LOG_WARNING for all the remain parts:

- Remove the defination of LOG_WARNING
- Use LOG_WARNING_F instead of LOG_WARNING
- Use LOG_WARNING_PREFIX instead of LOG_WARNING for class pegasus_server_impl 
- Remove some commentted LOG_WARNING code
- Some format fixes

### CODE REVIEW

File: src/runtime/task/task_worker.cpp [hunk 1]
```diff
@@ -103,7 +103,7 @@ void task_worker::set_name(const char *name)
 #endif // defined(__linux__)
     // We expect EPERM failures in sandboxed processes, just ignore those.
     if (err < 0 && errno != EPERM) {
-        LOG_WARNING("Fail to set pthread name. err = %d", err);
+        LOG_WARNING_F("Fail to set pthread name. err = {}", err);
```
Comment (empiredan) [inline src/runtime/task/task_worker.cpp:106]:
```suggestion
        LOG_WARNING_F("Fail to set pthread name: err = {}, msg = {}", err, utils::safe_strerror(errno));
```

File: src/runtime/task/task_worker.cpp [hunk 2]
```diff
@@ -128,7 +128,7 @@ void task_worker::set_priority(worker_priority_t pri)
         succ = false;
     }
     if (!succ) {
-        LOG_WARNING("You may need priviledge to set thread priority. errno = %d", errno);
+        LOG_WARNING_F("You may need priviledge to set thread priority. errno = {}", errno);
```
Comment (empiredan) [inline src/runtime/task/task_worker.cpp:131]:
```suggestion
        LOG_WARNING_F("You may need priviledge to set thread priority: errno = {}, msg = {}", errno, utils::safe_strerror(errno));
```

File: src/runtime/task/task_worker.cpp [hunk 3]
```diff
@@ -159,7 +159,7 @@ void task_worker::set_affinity(uint64_t affinity)
     err = pthread_setaffinity_np(pthread_self(), sizeof(cpuset), &cpuset);
 
     if (err != 0) {
-        LOG_WARNING("Fail to set thread affinity. err = %d", err);
+        LOG_WARNING_F("Fail to set thread affinity. err = {}", err);
```
Comment (empiredan) [inline src/runtime/task/task_worker.cpp:162]:
```suggestion
        LOG_WARNING_F("Fail to set thread affinity. err = {}, msg = {}", err, utils::safe_strerror(errno));
```

File: src/server/pegasus_server_impl.cpp [hunk 4]
```diff
@@ -438,24 +438,24 @@ void pegasus_server_impl::on_multi_get(multi_get_rpc rpc)
         if (c > 0 || (c == 0 && (!start_inclusive || !stop_inclusive))) {
             // empty sort key range
             if (_verbose_log) {
-                LOG_WARNING(
-                    "%s: empty sort key range for multi_get from %s: hash_key = \"%s\", "
-                    "start_sort_key = \"%s\" (%s), stop_sort_key = \"%s\" (%s), "
-                    "sort_key_filter_type = %s, sort_key_filter_pattern = \"%s\", "
-                    "final_start = \"%s\" (%s), final_stop = \"%s\" (%s)",
+                LOG_WARNING_F(
```
Comment (empiredan) [inline src/server/pegasus_server_impl.cpp:441]: Use `LOG_WARNING_PREFIX` instead:
```suggestion
                LOG_WARNING_PREFIX(
```

File: src/server/pegasus_server_impl.cpp [hunk 5]
```diff
@@ -438,24 +438,24 @@ void pegasus_server_impl::on_multi_get(multi_get_rpc rpc)
...
+                    "{}: empty sort key range for multi_get from {}: hash_key = \"{}\", "
```
Comment (empiredan) [inline src/server/pegasus_server_impl.cpp:442]:
```suggestion
                    "empty sort key range for multi_get from {}: hash_key = \"{}\", "
```

File: src/server/pegasus_server_impl.cpp [hunk 6]
```diff
@@ -438,24 +438,24 @@ void pegasus_server_impl::on_multi_get(multi_get_rpc rpc)
...
+                    "start_sort_key = \"{}\" ({}), stop_sort_key = \"{}\" ({}), "
+                    "sort_key_filter_type = {}, sort_key_filter_pattern = \"{}\", final_start = "
+                    "\"{}\" ({}), final_stop = \"{}\" ({})",
                     replica_name(),
```
Comment (empiredan) [inline src/server/pegasus_server_impl.cpp:446]:
```suggestion
```

**Review by empiredan — APPROVED** [review]

File: src/redis_protocol/proxy_lib/redis_parser.cpp [hunk 7]
```diff
@@ -899,7 +899,7 @@ void redis_parser::counter_internal(message_entry &entry)
     if (dsn::utils::iequals(command, "INCR") || dsn::utils::iequals(command, "DECR")) {
         if (entry.request.sub_requests.size() != 2) {
             LOG_WARNING_F("{}: command {} seqid({}) with invalid arguments count: {}",
-                          _remote_address.to_string(),
+                          _remote_address,
```
Comment (acelyc111) [inline src/redis_protocol/proxy_lib/redis_parser.cpp:902]: Nice!

**Review by acelyc111 — APPROVED** [review]

**Review by empiredan — APPROVED** [review]

**Review by acelyc111 — APPROVED** [review]

**Pull request merged**